src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const getMen = people.filter((person) => person.sex === 'm');
```

Comment:
the name is not great - `getMen` would fit a helper function that for expample processes array `people`
but this is just an array of `men`
```suggestion
  const men = people.filter((person) => person.sex === 'm');
```